Jenkinsfile_nightly

```diff
@@ -42,8 +42,24 @@ withNightlyPipeline(type, product, component) {
   env.TEST_URL = params.URL_TO_TEST
 
   after('checkout') {
-    sh 'curl -L https://github.com/git-lfs/git-lfs/releases/download/v2.7.1/git-lfs-linux-amd64-v2.7.1.tar.gz | tar xz -C ./bin git-lfs'
-    sh 'PATH=$PATH:./bin git lfs install --local && PATH=$PATH:./bin git lfs pull'
+      sh '''
+        set -e
+    
+        echo "7be85238cbbb957ab25de52b60279d40ba40d3faa72eeb2cb9fa77d6d92381e5  git-lfs-v2.7.1.tar.gz" > git-lfs-v2.7.1.checksum.txt
+        curl -L https://github.com/git-lfs/git-lfs/releases/download/v2.7.1/git-lfs-linux-amd64-v2.7.1.tar.gz > git-lfs-v2.7.1.tar.gz
+        sha256sum -c git-lfs-v2.7.1.checksum.txt
```

Comment:
Can you suppress output? There might be `--status` flag to return status code only.

Reply:
we could try something like https://unix.stackexchange.com/questions/119648/redirecting-to-dev-null/119650#119650 do you think this would be helpful though there is only minimal output

Reply:
Have you tried `--status` flag?

Reply:
where do you want the --status flag ?